samples/browser/src/main/java/org/mozilla/samples/browser/BrowserFragment.kt

```diff
@@ -117,15 +120,22 @@ class BrowserFragment : Fragment(), BackHandler {
             .addSearchProvider(
                 components.searchEngineManager.getDefaultSearchEngine(requireContext()),
                 components.searchUseCases.defaultSearch,
-                fetchClient = HttpURLConnectionClient(),
+                fetchClient = fetchClient,
                 mode = SearchSuggestionProvider.Mode.MULTIPLE_SUGGESTIONS)
             .addClipboardProvider(requireContext(), components.sessionUseCases.loadUrl)
 
         downloadsFeature.set(
             feature = DownloadsFeature(
-                requireContext(),
+                requireContext().applicationContext,
                 sessionManager = components.sessionManager,
                 fragmentManager = childFragmentManager,
+                onDownloadCompleted = { download ->
+                    Logger.debug("Download done. $download")
+                },
+                /*downloadManager = FetchDownloadManager(
```

Comment:
Is there any specific reason why we couldn't use `FetchDownloadManager` here?

Reply:
I didn't want to switch yet until the new download manager is more mature.

Reply:
👍